### PR TITLE
8.0 website blog share

### PR DESCRIPTION
**Pull request by andhit-r**
This module will add functionality to enable/disabled share button on each blog post. This module will be useful if user would like to have share button on all blog post. Without this module share button have to be added manually via share button snippet one by one.

Share button will appear after blog subtitle

### CODE REVIEW

**Comment by andhit-r** [comment]
@pedrobaeza @elicoidal @oscarolar would you kindly review this PR. Quite similar to https://github.com/OCA/e-commerce/pull/98.

File: website_blog_share/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```
Comment (pedrobaeza) [inline website_blog_share/README.rst]: Use latest README template

File: website_blog_share/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+1. Go to website menu
+2. Open one of your blog post
+3. Click on customize button
+4. Toggle share blog option to enable/disable share button
+
+NB: even if you have to go to one specific blog post, the set up is valid for all of them.
+
+Know issues / Roadmap
+====================
+* Share product configuration valid for all products. No individual configuration for each product
```
Comment (oscarolar) [inline website_blog_share/README.rst]: Is this **Known issues / Roadmap** section ok?,
Its seems to be a copy of the other PR you made before https://github.com/OCA/e-commerce/pull/98/files#diff-64255abdcd1527c8462793394b6af4c1R21, this is a description talking about products and this PR references to Blog posts.

**Comment by andhit-r** [comment]
Yes @oscarolar. It's my bad. Revised it. Now it's ready to review.

**Comment by oscarolar** [comment]
:+1:

**Comment by andhit-r** [comment]
@rafaelbn would you kindly put need reviews label for this PR. 

@pedrobaeza this PR is ready to be reviewed again.

Thanks before.

**Comment by pedrobaeza** [comment]
It misses the "Try me on runbot" paragraph on Usage section, but apart from that, :+1:

**Comment by andhit-r** [comment]
Runbot link added.

**Comment by max3903** [comment]
:+1:

**Pull request merged**